index.js

```diff
@@ -15,11 +16,20 @@ var nameMap = {
 };
 
 module.exports = function (release) {
-	var version = /\d+\.\d+/.exec(release || os.release());
+	var verRe = /\d+\.\d+/;
+	var version;
 
-	if (!version) {
+	if (release || !semver.satisfies(process.version, '>=0.12.0 <3.1.0')) {
+		version = verRe.exec(release || os.release());
+	} else if (/^win/i.test(process.platform)) {
```

Comment:
`} else if (process.platform === 'win32) {`

It's the same platform keyword for both 32 and 64-bit.